scripts/umount.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+. "$(dirname ${BASH_SOURCE[0]})/conf.cfg"
+
+umount "$MNT_DIR"
+cryptsetup close "$MAPPER_DEV"
+losetup -d `cat $LOOP_DEV_FILE`
+rm "$LOOP_DEV_FILE"
```

Comment:
Use either `"$LOOP_DEV_FILE"` or `$LOOP_DEV_FILE` in both cases, the former should be safer.

Reply:
Done